### ISSUE TITLE
on Poloniex, fetchBalance return price of GAS not USDT.

### ISSUE THREAD

**Issue by ntapp** [ccxt/ccxt#707]
this return is GAS balance.

```
  USDT: { free: 1.19052226, used: 0, total: 1.19052226 },
```

**Comment by kroitor** [comment]
@ntapp thanks for reporting this! Will take a look into it asap!

**Comment by kroitor** [comment]
@ntapp are you sure about this? Can you post verbose output of it?

The reply from poloniex, unparsed:

```
Array
(
    [info] => Array
        (
            [GAS] => Array
                (
                    [available] => 0.00000000
                    [onOrders] => 0.00000000
                    [btcValue] => 0.00000000
                )

            [USDT] => Array
                (
                    [available] => 0.03209058
                    [onOrders] => 0.00000000
                    [btcValue] => 0.00000293
                )
        )
```

Parsed info from ccxt:

```

    [GAS] => Array
        (
            [free] => 0
            [used] => 0
            [total] => 0
        )

    [USDT] => Array
        (
            [free] => 0.03209058
            [used] => 0
            [total] => 0.03209058
        )

    [free] => Array
        (
            [GAS] => 0
            [USDT] => 0.03209058
        )

    [used] => Array
        (
            [GAS] => 0
            [USDT] => 0
        )

    [total] => Array
        (
            [GAS] => 0
            [USDT] => 0.03209058
        )

)
```

<img width="1228" alt="screen shot 2017-11-29 at 15 50 33" src="https://user-images.githubusercontent.com/1294454/33376007-15d944be-d51d-11e7-8b18-5a763c0e13f7.png">

Seems correct to me.
